src/Twig/SentryExtension.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sentry\SentryBundle\Twig;
+
+use Sentry\State\HubInterface;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+final class SentryExtension extends AbstractExtension
+{
+    /**
+     * @var HubInterface The current hub
+     */
+    private $hub;
+
+    /**
+     * @param HubInterface $hub The current hub
+     */
+    public function __construct(HubInterface $hub)
+    {
+        $this->hub = $hub;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFunctions(): array
+    {
+        return [
+            new TwigFunction('sentry_trace_meta', \Closure::fromCallable([$this, 'getTraceMeta']), ['is_safe' => ['html']]),
+        ];
+    }
+
+    /**
+     * Returns an HTML meta tag named `sentry-trace`.
+     */
+    private function getTraceMeta(): string
+    {
+        $span = $this->hub->getSpan();
+
+        return sprintf('<meta name="sentry-trace" content="%s" />', null !== $span ? $span->toTraceparent() : '');
```

Comment:
I know it's a nitpick, but I had to double take because it seemed "transparent" :sweat_smile: 
```suggestion
        return sprintf('<meta name="sentry-trace" content="%s" />', null !== $span ? $span->toTraceParent() : '');
```

Reply:
Although in PHP the function names are case-insensitive, I think it's an error to call them with another casing

Reply:
Ok, so I think we should fix this with a PR, it's not a BC.